src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public const int SseCbAlign = 16;
```

Comment:
What does `CbAlign` stand for (count of bytes alignment)? 

Following .NET Naming Conventions might be desirable...

Reply:
I also want to know what `CbAlign` stands for.  I was following the existing code:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.CpuMath/Sse.cs#L11-L14

Reply:
After some detective work, the convention seems to stem from `AlignedArray.cs`:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.CpuMath/AlignedArray.cs#L19-L28

Reply:
`Cb` normally stands for something like `count of bytes`.

I would probably rename this to something like `Vector128Alignment`, or something similar. That makes it clear that it is the alignment for the `Vector128<T>` type, and follows a "normal" .NET Naming Convention.

Reply:
Changed the naming of `SseCbAlign` to `Vector128Alignment` in the latest commit.  Thanks for your suggestion!